en/app.ftl

```diff
@@ -278,6 +278,25 @@ landing-use-cases-signups-hero-content2 = Before you complete that next signup,
 landing-faq-headline = Top questions about { -brand-name-firefox-relay }
 landing-faq-cta = See more FAQs about { -brand-name-firefox-relay }
 
+## Landing Reviews Section
+
+landing-reviews-add-ons = Add-ons
+landing-reviews-logo-title = { -brand-name-firefox } Browser
```

Comment:
Line 15 (beginning of the doc) has an entry for `Firefox Browser`.

Reply:
We tried using that and it won't render on our end.

Reply:
We probably should check why it is not rendering or we will have non-standard entries. If we were to fix it later, it requires new ID and all locales have to rework it. For this one, it will be just a placeholder change, but in principle, we should look into why it is not working.

Reply:
@lloan I think @peiying2 means doing this:

```suggestion
landing-reviews-logo-title = { -brand-name-firefox-browser }
```